src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -24,6 +28,11 @@ internal void InitializeLicense(string licenseText)
 
             if (string.IsNullOrWhiteSpace(licenseText))
             {
+                if (FirstTimeUser() && (Debugger.IsAttached && SystemInformation.UserInteractive))
+                {
+                    TrackFirstTimeUsageEvent();
+                    SetupFirstTimeRegistryKeys();
```

Comment:
Btw. the previous code did` SendAsync().ContinueWith` which has also fire & forget semantics. So the problem is not related to `async void`